plugins/pulp_docker/plugins/distributors/publish_steps.py

```diff
@@ -14,6 +14,9 @@
 from pulp_docker.plugins.distributors import configuration, v1_publish_steps
 
 
+tags_names_redirect = {}
```

Comment:
I see. The risk of having anything in global namespace is that it will persist even when the task is done.

I suggest in ``V2WebPublisher.__init__``, make the ``dict``, and pass it as an argument to each of the steps that needs to use it.